### ISSUE TITLE
feat(patch): Fix descriptions

### ISSUE THREAD

**Issue by egscodes** [ReVanced/revanced-patches#1065]
### Application

Youtube

### Issue

Ok so some patches say that they disable some stuff, while other that say they disable they instead add a toggle

Basically to fix the description of some patches and if they add a toggle to say adds a toggle in there description.

Other thing that could be good is a image of what the patch actually does, like some descriptions do explain well but other are more confussing

### Patch

Ok so some patches say that they disable some stuff, while other that say they disable they instead add a toggle

basically to fix the description of some patches and if they add a toggle to say adds a toggle 

### Motivation

It will be less confussing

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [ ] I have written a short but informative title.
- [ ] I filled out all of the requested information in this issue properly.
